[2025-04-09T19:53:48.332+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_pipeline_modo_prueba.load_to_sqlite manual__2025-04-09T19:46:50.371176+00:00 [queued]>
[2025-04-09T19:53:48.344+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_pipeline_modo_prueba.load_to_sqlite manual__2025-04-09T19:46:50.371176+00:00 [queued]>
[2025-04-09T19:53:48.344+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-04-09T19:53:48.385+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): load_to_sqlite> on 2025-04-09 19:46:50.371176+00:00
[2025-04-09T19:53:48.391+0000] {standard_task_runner.py:60} INFO - Started process 204 to run task
[2025-04-09T19:53:48.395+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'etl_pipeline_modo_prueba', 'load_to_sqlite', 'manual__2025-04-09T19:46:50.371176+00:00', '--job-id', '164', '--raw', '--subdir', 'DAGS_FOLDER/etl_workflow.py', '--cfg-path', '/tmp/tmpyz1gpnhc']
[2025-04-09T19:53:48.396+0000] {standard_task_runner.py:88} INFO - Job 164: Subtask load_to_sqlite
[2025-04-09T19:53:48.483+0000] {task_command.py:423} INFO - Running <TaskInstance: etl_pipeline_modo_prueba.load_to_sqlite manual__2025-04-09T19:46:50.371176+00:00 [running]> on host 10542c8e3f93
[2025-04-09T19:53:48.640+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='simon' AIRFLOW_CTX_DAG_ID='etl_pipeline_modo_prueba' AIRFLOW_CTX_TASK_ID='load_to_sqlite' AIRFLOW_CTX_EXECUTION_DATE='2025-04-09T19:46:50.371176+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-09T19:46:50.371176+00:00'
[2025-04-09T19:53:48.641+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-04-09T19:53:48.642+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python3 /opt/***/scripts/load_to_sqlite.py']
[2025-04-09T19:53:48.657+0000] {subprocess.py:86} INFO - Output:
[2025-04-09T19:53:49.929+0000] {subprocess.py:93} INFO - âœ… Archivo parquet leÃ­do: /opt/***/data/processed/final_merged.parquet
[2025-04-09T19:53:49.930+0000] {subprocess.py:93} INFO - ðŸ”— Conectado a la base de datos SQLite en /opt/***/data/output/merged_data.db
[2025-04-09T19:53:49.930+0000] {subprocess.py:93} INFO - ðŸ“¦ Datos cargados en la tabla 'datos_finales'
[2025-04-09T19:53:49.930+0000] {subprocess.py:93} INFO - âœ… ConexiÃ³n cerrada. Proceso de carga completado.
[2025-04-09T19:53:50.031+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-04-09T19:53:50.150+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=etl_pipeline_modo_prueba, task_id=load_to_sqlite, execution_date=20250409T194650, start_date=20250409T195348, end_date=20250409T195350
[2025-04-09T19:53:50.225+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-04-09T19:53:50.249+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
